tests/e2e/tests/e2e_happy_path/DevWorkspaceHappyPath.spec.ts

```diff
@@ -0,0 +1,35 @@
+// /*********************************************************************
+//  * Copyright (c) 2021 Red Hat, Inc.
+//  *
+//  * This program and the accompanying materials are made
+//  * available under the terms of the Eclipse Public License 2.0
+//  * which is available at https://www.eclipse.org/legal/epl-2.0/
+//  *
+//  * SPDX-License-Identifier: EPL-2.0
+//  **********************************************************************/
+
+import { e2eContainer } from '../../inversify.config';
+import { CLASSES } from '../../inversify.types';
+import { DriverHelper } from '../../utils/DriverHelper';
+import * as projectAndFileTests from '../../testsLibrary/ProjectAndFileTests';
+import { TestConstants } from '../..';
+
+const driverHelper: DriverHelper = e2eContainer.get(CLASSES.DriverHelper);
+
+suite('Workspace creation via factory url', async () => {
+
+    let factoryUrl : string = `${TestConstants.TS_SELENIUM_DEVWORKSPACE_URL}`;
+    const workspaceSampleName: string = 'web-nodejs-sample';
```

Comment:
Workspace name `web-nodejs-sample` could be misleading depending on which devworkspace devfile URL is set.

What if we name test workspace as `devworkspace-sample`, not linking to concrete supported language?

Reply:
For now this test will check workspace created from [flattened_theia-next.yaml](https://github.com/devfile/devworkspace-operator/blob/main/samples/flattened_theia-next.yaml) yaml. And it uses [web-nodejs-sample](https://github.com/devfile/devworkspace-operator/blob/f92b82c48bb26c6be2f3335a6714efef985708f2/samples/flattened_theia-next.yaml#L9).

Reply:
IF there is only one type of supported workspaces, I might make sense to express it in the test code, e.g.:
```
// This test checks only workspace created from "web-nodejs-sample" https://github.com/devfile/devworkspace-operator/blob/main/samples/flattened_theia-next.yaml.
```